Further PA Fixes 10-10-2018

Markkos1992 opened the pull request:
Includes Recentering and Minor Point Errors on I-579, I-279, US 19 TRUCK (Pittsburgh), PA 28, PA 28 TRUCK (Brookville), PA 66, PA 66 BUS (Greensburg), PA 830, and PA 950.
Other Changes:
I-279: Lined up with PA 28 (may be debatable) and I-79.
I-579: Added Centre Ave (CenAve) and lined up with PA 28.
US 19: Added McFarland Rd (McFRd_S), Highland Ave (HigAve), and Sewickley Oakmont Rd (SewOakRd)
US 19 TRUCK (Pittsburgh): Added McFarland Rd (McFRd) and Wenzell Ave (WenAve).
US 422/PA 28/PA 66 Interchange: Simplified to a single point. (See http://forum.travelmapping.net/index.php?topic=2725.0)
US 22: Added Cozy Inn Cut-Off (CozyInnCut) and Old William Penn Hwy (OldWPHwy_E)
US 322: Added 8thAve, Perry St (PerST), and C.L. School Rd(CLSchRd)
US 119: Added Mt Pleasant St (MtPleSt).
PA 28: Replaced GolfLinkRd with Pheasant Farm Rd (PheFarmRd). Replaced LauRunRd with Calhoun School Rd (CalSchRd). Added Shannondale Rd (ShaRd). Added and lined up with PA 949. Replaced SenTraRd with Mt Pleasant Rd (MouPleRd), removed +X8, and added Eagles Nest Rd (EagNestRd) and Beechton Rd(BeeRd).
PA 66: Added Crestview Dr (CreDr). Replaced GolfLinkRd with Pheasant Farm Rd (PheFarmRd). Replaced LauRunRd with Calhoun School Rd (CalSchRd). Added Tylersburg Rd (TylRd), Arnold Ave (ArnAve), Guitonville Rd (GuiRd), and Blue Jay Creek Rd (BJCreRd).
PA 66 BUS(Greensburg): Added Mt Pleasant St (MtPleSt), also added PA130_W to reflect currency with PA 130 WB. Replaced McCDr with Locust Valley Rd (LocValRd).
PA 130: Added Locust Valley Rd (LocValRd), and PA66BUS_N to reflect PA 130 WB concurrency with PA 66 BUS.
PA 208: Replaced LicRd with Sunny Rd (SunRd)
PA 819: Added Mt Pleasant St (MtPleSt).
PA 830: (a very strange route IMO) Modified shaping to add Red Barn Rd, Stevenson Hill Rd (SteHillRd), and Beechtree Rd (BeeRd).
PA 948: Added Highland Rd (HigRd).
PA 949: Lined up with PA 28.
PA 950: Replaced CopRd with Pancoast Rd (PanRd)
US 422/US 222/PA 12 Interchange in Reading Minor Revision: http://forum.travelmapping.net/index.php?topic=2679.0

rickmastfan67 commented:
I'd put back that shaping point on I-579.  It was there due to the route being short.  It was always recommended on short routes to have those types of shaping points.  Removing it loses over .1 miles.

rickmastfan67 commented:
Also @mapcat, with adding all those new points to US-19 in the 'North Hills' of Pittsburgh, you missed one intersection that I believe would be a very important one to add.  The I-279 HOV Ramps intersection north of Cemetery Lane.  They are .4 miles from that intersection, and adding it also improves that sharp spike between the 'CemLn ' & 'TownRd' points that deviates a big way from US-19's true route in that area.

Markkos1992 commented:
@rickmastfan67: I made the following revisions:
 
I-579: Readding +X1. I also forgot to mention that I lined this road up with PA 380.
US 19: Added I-279 at HOV Lane access and Ivory Ave (IvoAve). Replaced WatBlvd with EastSt.
US 19 TRUCK (Pittsburgh): Added Nelson Run Rd (NelRunRd) and ViewAve
